timeless/poster/api.py

```diff
@@ -56,34 +57,42 @@ def send(self, **kwargs):
         return response
 
 
-class Authenticated(Poster):
+@attr.s
+class PosterAuthData:
+    """ Poster auth data class """
+    application_id = attr.ib()
+    application_secret = attr.ib()
+    redirect_uri = attr.ib()
+    code = attr.ib()
 
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
 
-    def access_token(self):
-        """Fetches authorization tokens
-        :return: Token retrieved from Poster api or error returned by poster pi
+class Authenticated:
+    """ Poster Auth class """
+    def __init__(self, auth_data: PosterAuthData, **kwargs):
+        self.auth_url = "https://joinposter.com/api/v2/auth/access_token"
```

Comment:
@vladarefiev let's move this to class attribute

Reply:
@vryazanov good catch, agree